release.config.js

```diff
@@ -15,7 +15,7 @@ git -C build/website/ add . || exit 1
 git -C build/website/ commit -m "chore: update website to version \${nextRelease.version}" || exit 2
 git -C build/website/ push || exit 3
 ./gradlew shadowJar --parallel || ./gradlew shadowJar --parallel || exit 4
-./gradlew uploadKotlinOSSRH uploadKotlinMultiplatform release --parallel || ./gradlew uploadKotlinOSSRH uploadKotlinMultiplatform release --parallel || ./gradlew uploadKotlinOSSRH uploadKotlinMultiplatform release --parallel exit 5
+./gradlew uploadKotlinOSSRH uploadKotlinMultiplatform uploadJvm uploadJs release --parallel || ./gradlew uploadKotlinOSSRH uploadKotlinMultiplatform uploadJvm uploadJs release --parallel || ./gradlew uploadKotlinOSSRH uploadKotlinMultiplatform uploadJvm uploadJs release --parallel exit 5
```

Comment:
OK this is getting waaaay to long
```suggestion
RELEASE_ON_CENTRAL="./gradlew uploadKotlinOSSRH uploadKotlinMultiplatform uploadJvm uploadJs release --parallel"
eval "$RELEASE_ON_CENTRAL" || eval "$RELEASE_ON_CENTRAL" || eval "$RELEASE_ON_CENTRAL" || exit 5
```

this is getting waaay too long otherwise... I hope it works als like this

Reply:
Is it ok if we try with this solution first? I think I like this solution a little bit more if I compare it to the task approach, as the root build.gradle.kts is already not-that-small and these commands are not used elsewhere.

Reply:
Yeah